Sources/Workspace/UserToolchain.swift

```diff
@@ -205,10 +205,17 @@ public final class UserToolchain: Toolchain {
         self.xctest = nil
       #endif
 
-        self.extraSwiftCFlags = (destination.target.isDarwin()
-                                    ? ["-sdk", destination.sdk.pathString]
-                                    : [])
-                                  + destination.extraSwiftCFlags
```

Comment:
There's a reference to `destination.extraSwiftCFlags` (as opposed to `self.extraSwiftCFlags`) on this line that I don't see in the new code. Was this intentional?